run.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/sh
 
-mkdir -p ${DOWNLOAD_DIR}
-gsutil cp ${SOURCE_PATH} ${DOWNLOAD_DIR}
+if [ ! -z "${DOWNLOAD_DIR}" ] && [ ! -d "${DOWNLOAD_DIR}" ]; then
+  mkdir -p "${DOWNLOAD_DIR}"
+fi
+if [ ! -z "${SOURCE_PATH}" ] && [ -d "${DOWNLOAD_DIR}" ]; then
+  gsutil cp "${SOURCE_PATH}" "${DOWNLOAD_DIR}" && exec crond -f -c /root/crontabs
```

Comment:
It seems that increase a crond process even if `exec crond -f` ran in the `crond`.